### PR TITLE
Let jenkins_validate see all prow jobs.

### PR DESCRIPTION
**Pull request by spxtr**

### CODE REVIEW

**Comment by krzyzacy** [comment]
do you want to add them to testgrid?

**Comment by spxtr** [comment]
Will do in the next PR.

**Comment by krzyzacy** [comment]
/lgtm

assume it would not create circular ref?

**Comment by spxtr** [comment]
> assume it would not create circular ref?

Nope.

File: prow/jobs/jobs_test.go [hunk 1]
```diff
@@ -246,3 +246,50 @@ func TestGetPresubmits(t *testing.T) {
...
+	for _, j1 := range expected {
+		found := false
+		for _, j2 := range actual {
+			if j1 == j2 {
+				found = true
+				break
+			}
+		}
+		if !found {
+			t.Errorf("Did not find job %s in output", expected)
```
Comment (krzyzacy) [inline prow/jobs/jobs_test.go]: j1?

Comment (spxtr) [inline prow/jobs/jobs_test.go]: Fixed.

**Comment by krzyzacy** [comment]
lgtm

**Pull request merged**